### PR TITLE
US30 Complete – Record a workout

### PR DESCRIPTION
**Pull request by znwilkins**
The completion of this story means that a user can record their workout, and save it to Firebase.

The recorded workout includes:
- The time of the workout
- The GPS coordinates of that workout, including when those coordinates were visited

### CODE REVIEW

File: app/src/release/res/values/google_maps_api.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    you will also need to set up a key for your release certificate.
+
+    Follow the directions here:
+
+    https://developers.google.com/maps/documentation/android/signup
+
+    Once you have your key (it starts with "AIza"), replace the "google_maps_key"
+    string in this file.
+    -->
+    <string name="google_maps_key" templateMergeStrategy="preserve" translatable="false">YOUR_KEY_HERE</string>
```
Comment (nickburris) [inline app/src/release/res/values/google_maps_api.xml]: Is this the same key that goes in `app/src/debug/res/values/google_maps_api.xml` ?

Comment (nickburris) [reply app/src/release/res/values/google_maps_api.xml]: Also, after we merge this pull request, I think it'd make sense to add both google_maps_api.xml files to gitignore so we can all put our API keys locally.

Comment (nickburris) [reply app/src/release/res/values/google_maps_api.xml]: Unless I'm completely misinterpreting this, and we should just all use the same key...

Comment (znwilkins) [reply app/src/release/res/values/google_maps_api.xml]: As discussed, this is a per app key, so no need to add to gitignore.

**Pull request merged**